packages/kbn-openapi-generator/src/template_service/register_helpers.ts

```diff
@@ -87,4 +87,6 @@ export function registerHelpers(handlebarsInstance: typeof Handlebars) {
       return circularRefs.has(`#/components/schemas/${schemaName}`);
     }
   );
+
+  handlebarsInstance.registerHelper('shouldCastExplicitly', ({ anyOf }) => anyOf?.length > 2);
```

Comment:
`oneOf` looks like a valid case as well.

nit: Having a comment why casting explicitly is required would be nice here.

Reply:
added support to oneOf for the future 👍 currently no schemas are using the same way as anyOf, I think oneOf is used mostly in the requestBody.